kythe/proto/analysis.proto

```diff
@@ -188,25 +188,27 @@ message KzipInfo {
     // The count of data present in compilation units for a given corpus and
     // language.
     message CompilationUnits {
+      reserved 2, 3;
+
       // Number of compilation units that have a specific corpus in their VName.
       int32 count = 1;
-      // Number of source files mentioned in these compilation units.
-      int32 sources = 2;
-      // Number of required inputs mentioned in these compilation units.
-      int32 required_inputs = 3;
     }
 
     // Map from compilation unit language to stats about compilation units with
     // that language.
     map<string, CompilationUnits> language_compilation_units = 3;
 
     message RequiredInputs {
+      // TODO(salguarnieri) Add count of unique inputs for this corpus?
```

Comment:
If we de-duplicate the inputs to get a unique count, is there reason to keep the old counter around too?

If we only need the one number, could we replace the `RequiredInputs` messages with just an int?